tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -268,6 +268,13 @@ protected virtual ITraceSampler GetSampler(ImmutableTracerSettings settings)
             // simply overriden with the remote configuration. Instead, remote rules are merged with local rules,
             // with remote rules taking precedence.
 
+            if (settings.AppsecStandaloneEnabledInternal)
+            {
+                var samplerStandalone = new TraceSampler(new TracerRateLimiter(1, 60_000));
```

Comment:
nit: we generally try to add names to raw values/bools, particularly when there's more than one
```suggestion
                var samplerStandalone = new TraceSampler(new TracerRateLimiter(maxTracesPerInterval: 1, intervalMilliseconds: 60_000));
```